### PR TITLE
[Logs] Updates Logpush instructions to match new flow

### PR DESCRIPTION
**Pull request by angelampcosta**
### Summary

Updates Logpush instructions to reflect the new setup flow

### CODE REVIEW

**Review by soheiokamoto — APPROVED** [review]

File: content/logs/get-started/enable-destinations/azure/index.md [hunk 1]
```diff
@@ -12,27 +12,32 @@ Cloudflare Logpush supports pushing logs directly to Microsoft Azure via the Clo
...
+    - **Path** - bucket location within the storage container
+    - **Organize logs into daily subfolders** (recommended) 
 
-8. Enter or select the following destination information:
+When you are done entering the destination details, select **Continue**.
 
-    - **SAS URL**
-    - **Blob container subpath (optional)**
-    - **Daily subfolders**
+7. To prove ownership, Cloudflare will send a file to your designated destination. To find the token, select the **Open** button in the **Overview** tab of the ownership challenge file, then paste it into the Cloudflare dashboard to verify your access to the bucket. Enter the **Ownership Token** and select **Continue**.
```
Comment (marciocloudflare) [inline content/logs/get-started/enable-destinations/azure/index.md:26]: From here on out, this is really similar to what you have for AWS. Maybe it's a good candidate to have in a partial file. This way you don't have to repeat yourself :)

File: content/logs/get-started/enable-destinations/google-cloud-storage/index.md [hunk 2]
```diff
@@ -12,27 +12,33 @@ Cloudflare Logpush supports pushing logs directly to Google Cloud Storage (GCS)
...
-    - **Bucket path**
-    - **Daily subfolders**
-    - For **Grant Cloudflare access to upload files to your bucket**, make sure your bucket has added Cloudflare's IAM as a user (if you did not add it already).
+7. To prove ownership, Cloudflare will send a file to your designated destination. To find the token, select the **Open** button in the **Overview** tab of the ownership challenge file, then paste it into the Cloudflare dashboard to verify your access to the bucket. Enter the **Ownership Token** and select **Continue**. 
 
-9. Select **Validate access**.
+8. Select the dataset to push to the storage service.
 
-10. Enter the **Ownership token** (included in a file or log Cloudflare sends to your provider) and select **Prove ownership**. To find the ownership token, select **Open** in the **Overview** tab of the ownership challenge file.
+9. In the next step, you need to configure your logpush job:
```
Comment (marciocloudflare) [inline content/logs/get-started/enable-destinations/google-cloud-storage/index.md:31]: Steps 9, 10 and 11 are really good candidates for a partial file. They seem to be the same for all services (AWS, Azure, GCP, etc)

File: content/logs/get-started/enable-destinations/http/index.md [hunk 3]
```diff
@@ -10,6 +10,28 @@ Cloudflare Logpush now supports the ability to send logs to configurable HTTP en
 
 Note that when using Logpush to HTTP endpoints, Cloudflare customers are expected to perform their own authentication of the pushed logs. For example, customers may specify a secret token in the URL or an HTTP header of the Logpush destination.
 
+## Manage via the Cloudflare dashboard
+
+{{<render file="_enable-logpush-job.md">}}
+
+5. In **Select a destination**, choose **HTTP destination**.
+
+6. Enter the **HTTP endpoint** where you want to send the logs and select **Continue**.
```
Comment (marciocloudflare) [inline content/logs/get-started/enable-destinations/http/index.md]:
```suggestion
6. Enter the **HTTP endpoint** where you want to send the logs to, and select **Continue**.
```

File: content/logs/get-started/enable-destinations/s3-compatible-endpoints/index.md [hunk 4]
```diff
@@ -23,27 +23,34 @@ For more information about Logpush and the current production APIs, refer to [Cl
...
-
-8. Enter or select the following destination information:
+5.  In **Select a destination**, choose **S3-Compatible**.
 
-    - **S3 Compatible Bucket Path**
+6. Enter or select the following destination information:
+    - **Bucket** - S3 Compatible bucket name
+    - **Path** - bucket location within the storage container
+    - **Organize logs into daily subfolders** (recommended) 
+    - **Endpoint URL** - The url without the bucket name or path. Example, `sfo2.digitaloceanspaces.com`.
```
Comment (marciocloudflare) [inline content/logs/get-started/enable-destinations/s3-compatible-endpoints/index.md]:
```suggestion
    - **Endpoint URL** - The URL without the bucket name or path. Example, `sfo2.digitaloceanspaces.com`.
```

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**